packages/monaco/src/browser/style/index.css

```diff
@@ -104,3 +104,7 @@
     outline-offset: calc(-1*var(--theia-border-width));
     outline-color: var(--theia-focusBorder);
 }
+
+.monaco-editor .rename-box.preview input {
+    color: var(--theia-editor-foreground);
+}
```

Comment:
@danarad05 can you update the standard `rename` as well, currently when the preference is off the display is incorrect:

![rename-pref-off](https://user-images.githubusercontent.com/40359487/100891828-aea1ca00-3487-11eb-93c1-45fa7f0990bd.gif)

Reply:
@vince-fugnitto 
Fixed. Please review again. 
Thanks